src/absil/il.fs

```diff
@@ -1552,7 +1557,7 @@ and [<Sealed>] ILTypeDefs(f : unit -> (string list * string * ILAttributes * Laz
     let mutable array = InlineDelayInit<_>(f)
     let mutable dict = InlineDelayInit<_>(fun () -> 
             let arr = array.Value
-            let t = Dictionary<_,_>(HashIdentity.Structural)
+            let t = Dictionary<_,_>(3, HashIdentity.Structural)
```

Comment:
Main reason is Fable (and Javascript maps/sets) have no proper support for equality comparers. Technically map/set trees with IComparer can be used as a substitute, but probably the performance will not be too good (not sure how important collection performance is here though). Very few places in FCS actually use a real equality comparer though. Technically this should go away as it's just a deficiency in Fable.